### ISSUE TITLE
WES - Move DNA

### ISSUE THREAD

**Issue by MattiasJanssonviapath** [moka-guys/helpdesk#288]
Hi guys,

A DNA sample was registered to the wrong patient and is now stuck on the wrong case in Moka.

Can you remove DNA 205744 from PRU457846:01 so I can add it to PRU457846:03?

Cheers, /Matt

**Comment by aledj2** [comment]
Hi Matt, 
I've removed the DNA from 01 so you should now be able to add it to 03. Can you let me know if it works!

On an slightly different matter , whilst investigating this it looked like a new WES test had been created for 01, which was then failed. In the future, if any tests are to be closed can they be set to not required rather than failed please.

Thanks,

**Comment by MattiasJanssonviapath** [comment]
Hi Aled,

Thanks, it worked! DNA is now connected to the correct case.

Regarding the failed case, ‘not required’ prohibits you from entering a new request, while ‘test failed’ does not. The new DNA was entered for analysis on the wrong case before I caught it. I can change it to ‘not required’ now.

Cheers,

/Matt

**Comment by aledj2** [comment]
Do you mean a new WES test can't be created if there's an existing test with a status of not required? Is this a common scenario? I can raise a seperate ticket to address this?

**Comment by MattiasJanssonviapath** [comment]
Yes, exactly. This is not common, though, and when required we just stick it in failed. No need to spend time on it.
